internal/api/v2/jrpc.go

```diff
@@ -80,6 +80,10 @@ func NewJrpc(opts JrpcOptions) (*JrpcMethods, error) {
 		return nil, errors.New("local node specified but no client provided")
 	}
 
+	if opts.Config.DebugJSONRPC {
+		jsonrpc2.DebugMethodFunc = true
+	}
```

Comment:
Since this is global, maybe it should be located in `Node.Start` (`internal/node/node.go`) or `Program.Start` (`cmd/accumulated/program.go`)?